webapi/_test_data/test_route/texts/README.md

```diff
@@ -0,0 +1,31 @@
+# context
+These files are materials for test cases in `StandoffV1R2RSpec`.
+
+The `mapping*` files describe mapping used to translate the `xml` encoded text into Knora's Standoff format.
+
+# mappings
+
+## Letter
+Is the mapping used for lettter transcription
+
+- mapping file: `mappingForLetter.xml`
+- example text files: `letter.xml`, `letter2.xml`, `letter3.xml`
+
+## StandardHTML
+Is the mapping used to translate html markup using the default mapping `OntologyConstants.KnoraBase.StandardMapping`.
+
+- mapping file: `mappingForStandardHTML.xml`  
```

Comment:
Why would we need the file `mappingForStandardHTML.xml`? 

Do you want to say that this is the XML serialization of what is already in `standoff-data.ttl` as the `StandardMapping`?

Reply:
Should we move this information to the docs?

Reply:
exactly, this mapping xml file is what I used to generate.
Yes, I'll update the doc with the new list of supported tags and name this file as the base to generate the ttl.
- [ ]  update docs

Reply:
ah, I get it. You generated the turtle from this file :-)

Reply:
commit fe75a0c96f65f35fa795627cb8b5b13670757429